buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -126,6 +126,16 @@ object Libs {
         name = "jacodb-api-jvm",
         version = Versions.jacodb
     )
+    val jacodb_api_storage = dep(
+        group = jacodbPackage,
+        name = "jacodb-api-storage",
+        version = Versions.jacodb
+    )
+    val jacodb_storage = dep(
```

Comment:
Both `jacodb-storage` and `jacodb-core` contain `META-INF/services/org.jacodb.api.storage.ers.EntityRelationshipStorageSPI` SPI configuraration files. 

During normal build these files should be merged.

However, in `usvm-jvm-instrumentation/build.gradle.kts` in `instrumentationRunnerJar` there's a following line preventing the merge of SPI configuration files:
```kotlin
duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Reply:
Addressed in https://github.com/UnitTestBot/usvm/commit/ef6402b830620e55e3a7f1da4cb5d373e82c71f8